src/v2/Apps/Collect/Routes/Collect/Utils/getMetadata.ts

```diff
@@ -15,21 +15,19 @@ export type Medium =
 
 export type Color =
   | "black-and-white"
```

Comment:
Nonblocking:

Why do we have `Black and White` as a combined filtered option? I know that we specifically have up right now a highlight for the "Black" show, which primarily features all black pieces. So why would these not be their own individual color options?

Reply:
@The-Beez-Kneez - think B&W photography. Monochrome is a whole sub-world inside the art world.

Reply:
Yeah, what @damassi said is correct. That said, it's an interesting question, and we certainly could consider having black & white as separate filters (because there are plenty of works that have lots of black or white in them). 

The way this filter works right now is [it checks](https://github.com/artsy/gravity/blob/1baa64541f1f2f75c6005a38688928ad4e445636/app/models/util/color_distance.rb#L63) if a work is _only_ black and white. So a work shouldn't be able to return both "black & white" and "red", for example.

Worth considering if we should split out black/white filters that could be included with other colors. I might suggest we do so for `black` but not for `white` because a white filter would return lots of works that have white backgrounds. There are lots of works that are shot against black backgrounds or have white matting/framing; it's hard to know if that would be enough to really skew the filter balance (i.e. showing lots of colorful works that happen to be shot against white backgrounds, rendering the filter somewhat useless) but I think it's possible.

Reply:
👍 Posed the question to FX's PM and designer [in Slack](https://artsy.slack.com/archives/C9SATFLUU/p1646146804105089), will see what they say!